### PR TITLE
fix: test.sh should use the variable $_group

### PR DESCRIPTION
**Pull request by BYK**

### CODE REVIEW

**Review by chadwhitacre — CHANGES_REQUESTED** [review]
Yes! But we need to set the variables before using them, eh?

**Comment by BYK** [comment]
>Yes! But we need to set the variables before using them, eh?

I mean, I thought about that but seems like they are set elsewhere _magically_ somehow?

**Comment by chadwhitacre** [comment]
Here's where they are set under `install.sh`:

https://github.com/getsentry/onpremise/blob/dd5a7d430e6341315a98dd0ad8da8988d65bab12/install/_lib.sh#L20-L26

Does that cover us here tho? 🤔

File: install/_lib.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ test "${DEBUG:-}" && set -x
 log_file="sentry_install_log-`date +'%Y-%m-%d_%H-%M-%S'`.txt"
 exec &> >(tee -a "$log_file")
 
-# Work from the onpremise root, no matter which script is called from where.
+# Work from /install/ for install.sh, project root otherwise
```
Comment (chadwhitacre) [inline install/_lib.sh:8]:
```suggestion
# Work from /install/ for install.sh, undefined otherwise
```

**Review by chadwhitacre — CHANGES_REQUESTED** [review]
Comment tweaks

**Review by chadwhitacre — APPROVED** [review]
Sike.

**Comment by chadwhitacre** [comment]
Lol confusing.

**Pull request merged**